sway-lib-std/src/u256.sw

```diff
@@ -0,0 +1,85 @@
+library u256;
+
+use core::num::*;
```

Comment:
Imports should always be explicit even if the compiler only brings into scope the code that is used. This is for the sake of code quality for the developers.

Reply:
@Braqzen any thoughts on the syntax for this import: `use core::num::*;`
where I only need the functions defined on the `u64` type.

Reply:
Unfortunately I do not know of a way to directly import that content without using `*` therefore that should stay as it is unless there is another way to be explicit